src/base/bittorrent/torrentimpl.cpp

```diff
@@ -2863,6 +2863,11 @@ QString TorrentImpl::createMagnetURI() const
         ret += u"&dn=" + QString::fromLatin1(QUrl::toPercentEncoding(displayName));
     }
 
+    if (hasMetadata())
+    {
+        ret += u"&xl=" + QString::number(totalSize());
+    }    
```

Comment:
It shouldn't have trailing whitespaces (and also removed redundant curly braces along the way):
```suggestion
    if (hasMetadata())
        ret += u"&xl=" + QString::number(totalSize());
```

Reply:
Which trailing whitespaces? I used curly braces for consistency with the surrounding code (and also because I prefer to use them)

Reply:
In any case, you still have remove the trailing whitespaces in order to pass the CI checks.